url.bs

```diff
@@ -2800,15 +2800,11 @@ takes a list of name-value tuples <var>tuples</var>, with an optional <a for=/>e
    <var>tuple</var>'s name, the
    <a><code>application/x-www-form-urlencoded</code> percent-encode set</a>, and true.
 
-   <li><p>Let <var>value</var> be <var>tuple</var>'s value.
-
-   <li><p>If <var>value</var> is a file, then set <var>value</var> to <var>value</var>'s filename.
-
-   <li><p>Set <var>value</var> to the result of running
-   <a for=string>percent-encode after encoding</a> with <var>encoding</var>, <var>value</var>, the
+   <li><p>Let <var>value</var> be the result of running <a for=string>percent-encode after
+   encoding</a> with <var>encoding</var>, <var>value</var>, the
```

Comment:
This specification (tries to) avoid(s) newlines inside inline elements. Also, you need to replace `<var>value</var>` with `<var>tuple</var>'s value`.